ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -0,0 +1,180 @@
+    }
+    const { complianceStandards } = data;
+    const modifiedData = data.controls.results.map((result) => {
+        const aggregationName = result?.aggregationKeys[0]?.id;
+        const standard = complianceStandards.find((cs) => cs.id === aggregationName);
+        const { numPassing, numFailing } = result;
+        const standardQueryValue =
+            standardLabels[standard?.id] || aggregationName || 'Unrecognized standard';
+        const query = getUrlQueryStringForSearchFilter({
+            ...searchFilter,
+            Cluster: searchFilter.Cluster || '*',
+            standard: standardQueryValue,
+        });
+        const link = `${complianceBasePath}/${
+            urlEntityListTypes[standardEntityTypes.CONTROL]
+        }?${query}`;
+        const modifiedResult = {
+            name: standard?.name || aggregationName || 'Unrecognized standard',
+            passing: Math.round((numPassing / (numFailing + numPassing)) * 100) || 0,
+            link,
+        };
+        return modifiedResult;
+    });
+
+    const sorted = sortBy(modifiedData, [(datum) => datum.passing]);
+
+    if (sortDirection === 'asc') {
+        sorted.reverse();
+    }
+
+    return sorted;
+}
+
+type AggregationResult = {
+    controls: {
+        results: {
+            aggregationKeys: {
+                id: keyof typeof standardTypes;
+                scope: StandardEntityType;
+            }[];
+            numFailing: number;
+            numPassing: number;
+            numSkipped: number;
+            unit: StandardEntityType;
+        }[];
+    };
+    complianceStandards: {
+        id: string;
+        name: string;
+    }[];
+};
+
+type SortBy = 'asc' | 'desc';
+
+function ComplianceLevelsByStandard() {
+    const { isOpen: isOptionsOpen, onToggle: toggleOptionsOpen } = useSelectToggle();
+    const { searchFilter } = useURLSearch();
+    const [sortDataBy, setSortDataBy] = useState<SortBy>('asc');
+
+    const where = getRequestQueryStringForSearchFilter({
+        // We always need to include some value for Cluster, otherwise aggregation will be performed at the namespace level
+        ...searchFilter,
+        Cluster: searchFilter.Cluster || '*',
+    });
+    const variables = {
+        groupBy: [entityTypes.STANDARD],
+        where,
+    };
+    const { loading, error, data, previousData } = useQuery<AggregationResult>(
+        AGGREGATED_RESULTS_ACROSS_ENTITIES,
+        { variables }
+    );
+
+    const complianceData = useMemo(
+        () => processData(searchFilter, sortDataBy, data || previousData)?.slice(0, 6),
+        [searchFilter, sortDataBy, data, previousData]
+    );
+
+    return (
+        <WidgetCard
+            isLoading={loading && !complianceData}
+            error={error}
+            header={
+                <Flex direction={{ default: 'row' }}>
+                    <FlexItem grow={{ default: 'grow' }}>
+                        <Title headingLevel="h2">Compliance coverage by standard</Title>
```

Comment:
The view does look a bit odd because of the long text here

<img width="572" alt="Screen Shot 2022-06-29 at 3 17 54 PM" src="https://user-images.githubusercontent.com/4805485/176555217-a01906b8-da88-475c-9469-2050a1390794.png">

Some solutions I thought might help would be:
1. See if UX can provide a shorter title
2. Wrap the title text and keep the actions in the top-right corner 

cc: @alanonthegit

Reply:
There's fixes for this that do provide a shorter title, as well as a better implementation of the grid in an upstream PR as well. :)